src/app/(auth)/authModel.style.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  Modal,
+  Box,
+  ButtonBase,
+  Typography,
+  Backdrop,
+  styled,
+  TextField,
+} from '@mui/material';
+import Link from 'next/link';
+// Style for the Backdrop
+export const StyledBackdrop = styled(Backdrop)({
+  backgroundColor: 'rgba(255, 255, 255, 0.1)',
+  backdropFilter: 'blur(7.8px)',
+});
+
+export const AuthModalContainer = styled(Modal)({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+export const AuthModalContent = styled(Box)({
+  backgroundColor: '#fff',
+  padding: '69px 49px',
+  borderRadius: '20px',
+  width: '347px',
+  boxShadow: '0 4px 20px rgba(0, 0, 0, 0.1)',
+  zIndex: 1,
+});
+export const StyledTextField = styled(TextField)({
+  '& .MuiInputBase-input': {
+    fontWeight: 600,
+    fontSize: '14px',
+  },
+  '& .MuiInputLabel-root': {
+    color: '#818181',
+    fontWeight: 600,
+    fontSize: '10px',
+    textTransform: 'capitalize',
+    '& .Mui-focused': {
+      color: '#818181',
+    },
+  },
+  '& .MuiInputLabel-root.Mui-focused': {
```

Comment:
The form elements in an application are generally all the same used everywhere, move these into `@/components/Form.styles.ts` so that they can be used appwide